backend/src/org/commcare/util/CommCareSession.java

```diff
@@ -721,6 +721,21 @@ public boolean isViewCommand(String command) {
         // commcare core for release issues
         return (entries.size() == 1 &&
                 (prototype.getXFormNamespace() == null ||
-                        prototype.getXFormNamespace().equals("")));
+                        prototype.getXFormNamespace().equals("")) &&
```

Comment:
In my opinion it is good to call "equals" on the concrete string to avoid null pointer exceptions. For instance "".equals(prototype.getXFormNamespace())